firestore/src/jni/object_arena.cc

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "firestore/src/jni/object_arena.h"
+
+#include "firestore/src/jni/env.h"
+#include "firestore/src/jni/hash_map.h"
+#include "firestore/src/jni/loader.h"
+#include "firestore/src/jni/long.h"
+
+namespace firebase {
+namespace firestore {
+namespace jni {
+namespace {
+
+Global<ObjectArena> kInstance;
```

Comment:
kInstance and kHashMap are not initialized anywhere. They should be initialized in ObjectArena::Initialize().